GC.md

```diff
@@ -0,0 +1,207 @@
+NIP-GC
+======
+
+Geocaching Events
+-----------------
+
+`draft` `optional`
+
+This NIP defines event kinds for geocaching activities on Nostr, enabling users to create, discover, and log geocaches in a decentralized manner.
+
+## Motivation
+
+Geocaching is a global treasure hunting game where participants use GPS coordinates to hide and seek containers ("geocaches") at specific locations. This NIP brings geocaching to Nostr, allowing for censorship-resistant, decentralized geocache listings and logs without relying on centralized platforms.
+
+## Event Kinds
+
+This NIP defines two new event kinds:
+
+- `kind:37515`: Geocache listing (addressable event)
+- `kind:37516`: Geocache log entry
```

Comment:
these should be normal kinds, 7515 and 7516 will work much better

Reply:
I think replaceable makes more sense in this case

Reply:
why?

Reply:
A few of the tags seem intended to change, particularly status, but maybe also hint, image, t. I agree that the log entries should be immutable.

Reply:
"status" specifically seems to be useless. a log entry will probably work much better, either by the original poster or by someone else.

if you're interested in a geocache you'll likely click on it to fetch its history, and maybe it was abandoned by the original poster but you see that others have reported going to the place and finding nothing, or something like that, then you know it's abandoned.

I think it might be a good idea to create another kind for "status report" or something like that.